waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/model/PrefixingMetastoreFilter.java

```diff
@@ -33,9 +33,12 @@
  * */
 public class PrefixingMetastoreFilter implements MetaStoreFilterHook {
 
-  public static final String PREFIX = "prefix-";
+  public static final String PREFIX_KEY = "hive.metastore.hooks.prefix";
+  public static final String PREFIX_DEFAULT = "prefix-";
+  private final String prefix;
 
   public PrefixingMetastoreFilter(HiveConf conf) {
+    prefix = conf.get(PREFIX_KEY,PREFIX_DEFAULT);
```

Comment:
Because of https://github.com/ExpediaGroup/waggle-dance/pull/322/commits/ab446906534087e45fa4683d079357fc8207b3a8 and https://github.com/ExpediaGroup/waggle-dance/pull/322/commits/0d67f268fc481aae5c02818321f7e1d229e6d85b, I think it's okay